core/services/offchainreporting/run_saver.go

```diff
@@ -39,14 +39,14 @@ func (r *RunResultSaver) Start() error {
 			for {
 				select {
 				case rr := <-r.runResults:
-					logger.Debugw("RunSaver: saving job run", "run", rr.Run, "task results", rr.TaskRunResults)
+					r.logger.Infow("RunSaver: saving job run", "run", rr.Run, "task results", rr.TaskRunResults)
```

Comment:
Why the upgrade to Info? Is this related to the request from @tyrion70 ?

Reply:
Yeah, it's on the linked ticket